content/image.js

```diff
@@ -45,6 +45,24 @@ function findByURL(url) {
   return find(urlToFilePath(url));
 }
 
+function findByURLWithFallback(url) {
+  let filePath = findByURL(url);
+  const locale = url.split("/")[1].toLowerCase();
+  if (
+    !filePath &&
+    locale !== DEFAULT_LOCALE &&
+    !url.startsWith(`/${DEFAULT_LOCALE.toLowerCase()}/`)
```

Comment:
Isn't this line redundant?

Reply:
I reused the existing check for static images which comes from https://github.com/mdn/yari/blob/49340d3c82f69b95f3c06804d9081725fec65ba0/build/check-images.js#L153-L157
However since the input here is the only URL, you're right. I'll be testing a bit and submit something.

Reply:
Changed :)